stable/artifactory-ha/CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+<<<<<<< HEAD
```

Comment:
@danielezer Can you fix this?

Reply:
Whoops, sorry I didn't commit my conflict resolutions. Sorry about that